graph/cytoscape/cytoscape.go

```diff
@@ -30,17 +31,18 @@ type NodeData struct {
 	Parent string `json:"parent,omitempty"` // Compound Node parent ID
 
 	// App Fields (not required by Cytoscape)
-	Service          string `json:"service"`
-	Version          string `json:"version,omitempty"`
-	Rate             string `json:"rate,omitempty"`             // edge aggregate
-	Rate3xx          string `json:"rate3XX,omitempty"`          // edge aggregate
-	Rate4xx          string `json:"rate4XX,omitempty"`          // edge aggregate
-	Rate5xx          string `json:"rate5XX,omitempty"`          // edge aggregate
-	IsCircuitBreaker string `json:"isCircuitBreaker,omitempty"` // true | false | unknown
-	IsGroup          string `json:"isGroup,omitempty"`          // set to the grouping type, current values: [ 'version' ]
-	IsRoot           string `json:"isRoot,omitempty"`           // true | false
-	IsSelfInvoke     string `json:"isSelfInvoke,omitempty"`     // rate of selfInvoke
-	IsUnused         string `json:"isUnused,omitempty"`         // true | false
+	Service             string                     `json:"service"`
+	Version             string                     `json:"version,omitempty"`
+	Rate                string                     `json:"rate,omitempty"`             // edge aggregate
+	Rate3xx             string                     `json:"rate3XX,omitempty"`          // edge aggregate
+	Rate4xx             string                     `json:"rate4XX,omitempty"`          // edge aggregate
+	Rate5xx             string                     `json:"rate5XX,omitempty"`          // edge aggregate
+	IsCircuitBreaker    string                     `json:"isCircuitBreaker,omitempty"` // true | false | unknown
```

Comment:
This name doesn't make sense (I know it isn't part of this PR but...). The node isn't a circuit breaker so saying "isCircuitBreaker==true" is misleading. It HAS a circuit breaker, but we do not say it IS a circuit breaker. If we want to keep these boolean-like attributes the same (that is, all prefixed with "Is") then at least add to the comment here to explain that the node isn't a kind of circuit breaker, but rather this denotes it HAS a circuit breaker in front of it.